roles/barbican/templates/etc/barbican/barbican.conf

```diff
@@ -0,0 +1,396 @@
+# {{ ansible_managed }}
+
+[DEFAULT]
+# Disable stderr logging
+use_stderr = False
+# Show debugging output in logs (sets DEBUG log level output)
+debug = False
+
+# Address to bind the API server
+bind_host = 0.0.0.0
+
+# Port to bind the API server to
+bind_port = 9311
```

Comment:
This has to be the backend_api port {{ endpoints.barbican.port.backend_api }}. Thats is where the service is running.